### PR TITLE
Docs/readme changes

### PR DESCRIPTION
**Pull request by kimerran**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,17 @@
...
-1. Clone this repo.
-2. Run `npm install`
-3. Run `npm run build`
-4. ???
-5. PROFIT!
+## how to contribute
+[Contributing guidelines](https://github.com/platf0rm/platform/blob/master/contributing-guidelines.md)
 
-## Development
+## api specs
```
Comment (ghost) [inline README.md]: Documentation is self-explanatory than API Spec

**Comment by ghost** [comment]
@kimerran LGTM

**Review by ghost — APPROVED** [review]

**Review by TheoryOfNekomata — APPROVED** [review]
For specs, perhaps we could use the Wiki instead?

**Comment by ghost** [comment]
@Temoto-kun it would be but it is nice to have a copy that could be easily cloned.

**Comment by TheoryOfNekomata** [comment]
@vkeusebio you can clone wikis. They are git repositories as well: http://stackoverflow.com/questions/18759738/effortless-export-from-github-wiki

**Comment by ghost** [comment]
@Temoto-kun thank you for pointing that out. @kimerran what do you think? should we move the docs to wiki instead?

**Comment by kimerran** [comment]
@vkeusebio we are still in the phase where nothing is finaly. I'm not seeing the need to build the wiki for now

**Pull request merged**